Podfile

```diff
@@ -7,15 +7,28 @@ target 'RInAppMessaging_Example' do
   pod 'RInAppMessaging', :path => '.'
   pod 'SwiftLint', '~> 0.42'
 
-  target 'Tests' do
+  abstract_target 'Tests-Common' do
     pod 'Quick'
     pod 'Nimble'
-  end
 
-  target 'IntegrationTests' do
+    target 'Tests' do
+    end
+
+    target 'UITests' do
+    end
+
+    target 'IntegrationTests' do
+    end
   end
 end
 
 post_install do |installer|
   system("./configure-secrets.sh InAppMessaging #{secrets.join(" ")}")
+  installer.pods_project.targets.each do |target|
+    target.build_configurations.each do |config|
+      if target.name == 'RInAppMessaging'
+        config.build_settings['ENABLE_TESTABILITY'] = 'YES'
```

Comment:
What's this for? [`ENABLE_TESTABILITY=YES` is already set for the parent `RInAppMessaging` project](https://github.com/rakutentech/ios-inappmessaging/blob/master/RInAppMessaging.xcodeproj/project.pbxproj#L702) then it should be inherited by your new `UITest` target

Reply:
RInAppMessaging target inherits from `Pods` project unfortunately.